pages/_lang/generate.vue

```diff
@@ -1134,7 +1151,32 @@ footer a {
   font-size: 14px;
   line-height: 21px;
 }
+#invalidUrlToast {
```

Comment:
When does this trigger? In preview, I just tried putting in "foo" and didn't get any invalid URL.